src/main/java/uk/ac/cam/cl/dtg/isaac/configuration/IsaacApplicationRegister.java

```diff
@@ -117,6 +105,9 @@ public IsaacApplicationRegister() {
             // initialise filters
             this.singletons.add(injector.getInstance(PerformanceMonitor.class));
             this.singletons.add(injector.getInstance(AuditMonitor.class));
+
+            // initialise observers
+            this.singletons.add(injector.getInstance(IGroupObserver.class));
```

Comment:
This was just bound as an EagerSingleton in the Isaac Guice config, but because the Segue Guice config is created many times, it would get created many times if I just moved the bind to the Segue one. Instead, make it have a provider method and create it as a singleton once here on startup.